frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -153,22 +153,23 @@ const osUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
   if (iGetCommonData(state, id, VMWizardProps.isProviderImport)) {
     return;
   }
+  if (iGetCommonData(state, id, VMWizardProps.isProviderImport)) {
```

Comment:
twice?

Reply:
wanted to be extra sure :-) , removed :+1: